docs/api-reference/COURSE-API.md

```diff
@@ -3,77 +3,43 @@
 
 # Course API
 
-### **get_term_courses(term, subject)**
+### **get_subject_courses(subject)**
 
 #### **Parameters**:
-  - `term`: Term  | Example: `2194`
   - `subject`: Subject Code | Example: `'CS'`
 
 #### **Returns**:
-Returns a `PittSubject` object containing the information of all courses and sections offered during the term.
+Returns a `Subject` object containing the information of all courses underneath that subject.
 #### **Example**:
 
 ###### **Sample Usage**:
 ```python
->>> cs_subject = course.get_courses(term='2194', subject='CS')
+>>> cs_subject = course.get_subject_courses(term='2194', subject='CS')
```

Comment:
If the function no longer takes `term` as an argument, this README line is incorrect.

Reply:
Speaking of which, what happened to the term information? How can I get it?

Reply:
> Speaking of which, what happened to the term information? How can I get it?

That function returns all courses that are listed on peoplesoft underneath that subject, regardless of what term they've been offered. To get course offerings for a specific term, that can be done with `get_course_details` which takes term as a parameter.